### PR TITLE
Add new async_loading field in INFO command

### PR DESCRIPTION
**Pull request by enjoy-binbin**
Added in https://github.com/redis/redis/pull/9323

### CODE REVIEW

**Comment by oranagra** [comment]
Thanks, indeed e should have documented it and forgot. 
But I think the text should be improved to add more details, I.e. State that in this mode redis is in the process of loading a new dataset, and but commands are allowed and the `loading` flag is off. Also it need sot refer to `repl-diskless-load swapdb`

File: commands/info.md [hunk 1]
```diff
@@ -172,6 +172,7 @@ by referring to the `MEMORY STATS` command and the `MEMORY DOCTOR`.
 Here is the meaning of all fields in the **persistence** section:
 
 *   `loading`: Flag indicating if the load of a dump file is on-going
+*   `async_loading`: To differentiate from the blocking loading - Added in Redis 7.0.
```
Comment (yoav-steinberg) [inline commands/info.md:175]:
```suggestion
*   `async_loading`: Currently busy loading replication data-set asynchronously while serving old data. This means `repl-diskless-load` is enabled and set to `swapdb`. Added in Redis 7.0.
```

**Review by yoav-steinberg — COMMENTED** [review]
@oranagra changed it based on your comments. Let me know if it makes sense.

File: commands/info.md (hunk 1 above)
Comment (oranagra) [reply commands/info.md:175]: LTGM, one suggestion: maybe drop the word "busy", (not to be confused with `-BUSY` error).
the text "currently loading..." would be enough..

**Pull request merged**